### PR TITLE
Feature/hello world

### PR DESCRIPTION
**Pull request by kensayo**
- Create backend with RoR
- API response with random message
- Front end created using React
- Redux for state management

### CODE REVIEW

**Comment by Davidosky007** [comment]
Hi @kensayo 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no pull request description 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+# rubocop:disable Lint/Syntax
+messages = Message.create([{ message: 'Hello World!' }, { message: 'Welcome my friend to React/Rails/Redux...', { message: 'Are you still here?' }, { message: 'Dont look me like that..' }, { message: 'Why are you here?' } }])
+# rubocop:enable Lint/Syntax
```
Comment (AshakaE) [inline db/seeds.rb]: - [ ] Kindly fix the syntax error in your db seed 😄

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  namespace :v1, defaults: { format: 'json' } do
+    get 'message', to: 'messages#index'
+  end
+  get '*page', to: 'statics#index', constraints: lambda { |req|
+    !req.xhr? && req.format.html?
+  }
+  root 'statics#index'
```
Comment (AshakaE) [inline config/routes.rb:8]: - [ ] Kindly fix the root page view , I'm getting this error
image

File: README.md [hunk 3]
```diff
@@ -1,24 +1,60 @@
...
+// Create the database
+rails db:create
+// Migrate the tables
+rails db:migrate
+// Populate the database
+rails db:seed
+// Start the server
+rails server
+```
+- Then just go to [http://127.0.0.1:3000/](http://127.0.0.1:3000/)
```
Comment (AshakaE) [inline README.md]: - [ ] Kindly update this link to take the user to the right greetings page

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @kensayo  🙋‍♀️, 

Good job so far! 👌

Your:
   - Project is impressive 👏 
   - Git flow ✔️ 
   - Pull request 👍 
   - README 👌 
   
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by enaburekhan — APPROVED** [review]
Hi Kensayo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**